libs/cx/side-navigation/side-navigation-utility.directive.ts

```diff
@@ -3,7 +3,8 @@ import { NestedListComponent } from '@fundamental-ngx/cx/nested-list';
 
 @Directive({
     // eslint-disable-next-line @angular-eslint/directive-selector
-    selector: '[cxSideNavigationUtility], [fdx-side-nav-utility]'
+    selector: '[cxSideNavigationUtility], [fdx-side-nav-utility]',
+    standalone: false
```

Comment:
can be removed

Reply:
no it can't, in angular 19 default for standalone is true